lib/mongo/server/description.rb

```diff
@@ -237,6 +243,10 @@ def initialize(address, config = {}, average_round_trip_time = nil)
       # @return [ Hash ] The actual result from the ismaster command.
       attr_reader :config
 
+      def load_balancer?
```

Comment:
Maybe a short docstring here? Since all other methods in this class have one.

Reply:
Added, there should've been one here.